homeassistant/components/microsoft_face/__init__.py

```diff
@@ -113,7 +113,7 @@
             face.store.pop(g_id)
 
             entity = entities.pop(g_id)
-            hass.states.async_remove(entity.entity_id)
+            hass.states.async_remove(entity.entity_id, service.context)
```

Comment:
Side note: This is weird that we remove the state but not the entity.

Reply:
It looks like this integration uses entities but not EntityComponent/Platform. So it is actually removed on line 115. I agree that it's weird…